src/dotnet/BuildServer/IBuildServerManager.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.DotNet.BuildServer
+{
+    internal interface IBuildServerManager
+    {
+        Result Shutdown();
```

Comment:
>Result Shutdown(); [](start = 8, length = 18)

Is there value in making this async? VBCScompiler and likely Razor can respond to this quite promptly as we're just going to kill the process. The MSBuild nodes though have to do cooperative shutdown which may take some amount of time.

Reply:
Sure, that should be easy enough to do.  I'll rethink the command output.